hack/local-up-cluster.sh

```diff
@@ -362,7 +364,7 @@ function start_kubelet {
         -i \
         --cidfile=$KUBELET_CIDFILE \
         gcr.io/google_containers/kubelet \
-        /kubelet --v=3 --containerized ${priv_arg}--chaos-chance="${CHAOS_CHANCE}" --hostname-override="${HOSTNAME_OVERRIDE}" --address="127.0.0.1" --api-servers="${API_HOST}:${API_PORT}" --port="$KUBELET_PORT" --resource-container="" &> $KUBELET_LOG &
+        /kubelet --v=3 --containerized ${priv_arg}--chaos-chance="${CHAOS_CHANCE}" --hostname-override="${HOSTNAME_OVERRIDE}" --address="127.0.0.1" --api-servers="${API_HOST}:${API_PORT}" --port="$KUBELET_PORT" --resource-container="" --sync-frequency=${KUBELET_SYNC_FREQUENCY} &> $KUBELET_LOG &
```

Comment:
Unfortunately, I don't think this flag work as you expected. kubelet cleans up the mounts every two second (the housekeeping interval).https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L2399 This is not configurable, but it should generally be sufficient for your 20 second wait time.